dev-util/ninja/ninja-1.11.1-r3.ebuild

```diff
@@ -0,0 +1,128 @@
+# Copyright 2012-2023 Gentoo Authors
```

Comment:
Do we want to add the pkg_* hack like we do for tar, gzip?

Reply:
I didn't add it because `app-alternatives/ninja` should be immediately pulled in after the upgrade, and ninja shouldn't be required to merge it.

Reply:
I think there's technically a risk that you'd be a running a parallel job that just happens to call `ninja` during the window.

Reply:
Sure.

Reply:
> I think there's technically a risk that you'd be a running a parallel job that just happens to call `ninja` during the window.

FYI it is very possible, see: https://bugs.gentoo.org/915980